lib/post_destroyer.rb

```diff
@@ -5,6 +5,8 @@
 # this class contains the logic to delete it.
 #
 class PostDestroyer
+  USER_STAT_COUNT_DECREMENT = "decrement"
```

Comment:
Why not just use `:decrement` and `:increment` symbols instead of string constants? I don't find `USER_STAT_COUNT_DECREMENT` to be clearer than `:decrement`. Quite the opposite actually. The verbosity is off-putting.